ansible/playbooks/AdoptOpenJDK_AIX_Playbook/group_vars/all/main.yml

```diff
@@ -15,12 +15,12 @@ Sudoers_File: /etc/sudoers
 
 # Jenkins User Variables:
 Jenkins_Username: jenkins
-Jenkins_User_SSHKey: /Vendor_Files/keys/id_rsa.pub
+Jenkins_User_SSHKey: /Vendor_Files/Jenkins_User_SSHKey
```

Comment:
I don't think these lines are going to achieve the desired effect (sadly) - the `_SSHKey` values now ned to be the contents of the file, not the filename itself

Reply:
Ah okay - I'll revert those, and I'll make a separate PR for AIX :-)